runtime/jcl/common/mgmtosext.c

```diff
@@ -96,7 +98,10 @@ Java_com_ibm_lang_management_internal_ExtendedOperatingSystemMXBeanImpl_getSyste
 			portLibraryStatus = -3;
 			break;
 		default:
-			portLibraryStatus = OMRPORT_ERROR_OPFAILED;
+			portLibraryStatus =
```

Comment:
were you able to verify that this case is only possible if it is the first call?

Reply:
Yes if the time tracking mechanism works correctly.

Reply:
In the case it doesnt work correctly we still need to return a negative number

Reply:
So `omrsysinfo_get_CPU_load()` need to return another error code?

Reply:
yes, if there is more than one way to return `OMRPORT_ERROR_OPFAILED`

Reply:
I've made a change to OMR to return another error code on first call https://github.com/eclipse/omr/pull/7189